lib/IAMClient.js

```diff
@@ -52,6 +53,19 @@ class IAMClient {
         }
         this.accessKey = accessKey;
         this.secretKeyValue = secretKeyValue;
+        this.log = new Logger('IAMClient', {
```

Comment:
Okay, I've renamed the class in this file and updated the logs and JSDocs accordingly.